bin/bedrock

```diff
@@ -83,11 +83,16 @@ function addCommand(command, descriptor) {
 }
 
 function getHandler(descriptor) {
-  const { name, parent } = descriptor;
+  const { functionName, name, parent } = descriptor;
   if (parent.name === 'bedrock') {
     return require(path.resolve(__dirname, '../src', name)).default;
   } else {
-    return require(path.resolve(__dirname, '../src', parent.name))[camelCase(name)];
+    const func = functionName || camelCase(name);
+    if (parent.parent && parent.parent.name !== 'bedrock') {
```

Comment:
I'm having trouble parsing this... is this just there because the `addCommand` function isn't as recursive as I thought it was?

Reply:
Yes, I just needed to go one folder deeper, e.g., `src/cloud/provision/` and `src/cloud/secret/`